dev/check_order_setup.py

```diff
@@ -0,0 +1,175 @@
+#!/usr/bin/env python
```

Comment:
Why did we change the location of this file?

Reply:
Because it's not a test really (at least not run by pytest) - I thought that rather than keeping it in test folder, I move it to dev. We do not seem to have  a good place for pre-commit checks like that, so I thought dev is ok . Any other ides where we can keep it ?

Reply:
I would probably in `scripts/ci` where we already have other scripts for precommits like others, Example:

```
❯ ls -ltr scripts/ci | grep pre_commit
-rwxr-xr-x@  1 kaxilnaik  staff   1105 18 May 20:12 pre_commit_bat_tests.sh
-rwxr-xr-x@  1 kaxilnaik  staff   1958 18 May 20:12 pre_commit_breeze_cmd_line.sh
-rwxr-xr-x@  1 kaxilnaik  staff   1136 18 May 20:12 pre_commit_check_license.sh
-rwxr-xr-x@  1 kaxilnaik  staff   1062 18 May 20:12 pre_commit_ci_build.sh
-rwxr-xr-x@  1 kaxilnaik  staff   1012 18 May 20:12 pre_commit_flake8.sh
-rwxr-xr-x@  1 kaxilnaik  staff   1077 18 May 20:12 pre_commit_generate_requirements.sh
-rwxr-xr-x@  1 kaxilnaik  staff    960 18 May 20:12 pre_commit_lint_dockerfile.sh
-rwxr-xr-x@  1 kaxilnaik  staff   1808 18 May 20:12 pre_commit_local_yml_mounts.sh
-rwxr-xr-x@  1 kaxilnaik  staff   1008 18 May 20:12 pre_commit_mypy.sh
-rwxr-xr-x@  1 kaxilnaik  staff   1088 18 May 20:12 pre_commit_update_extras.sh
-rwxr-xr-x@  1 kaxilnaik  staff   1474 21 May 17:06 pre_commit_check_integrations.sh
-rwxr-xr-x   1 kaxilnaik  staff   5321 23 May 02:17 pre_commit_yaml_to_cfg.py
```

. Things inside `dev` aren't really tested and currently mostly only contains Release related files

Reply:
LGTM other than that

Reply:
sure. I can move it there :)